e2e/nomostest/clusters/gke.go

```diff
@@ -85,7 +85,10 @@ func createGKECluster(t testing.NTB, name string) error {
 	} else {
 		args = append(args, "create")
 	}
-	args = append(args, name, "--project", *e2e.GCPProject)
+	args = append(args, name,
```

Comment:
do you want these to have fleet membership too?

Reply:
We don't configure that with cluster creation afaict. It looks like we register the fleet memberships in code here:
https://github.com/GoogleContainerTools/kpt-config-sync/blob/main/e2e/testcases/csr_auth_test.go#L394